### PR TITLE
Fix controls scope to only the current image

### LINKED ISSUE #6709: [Imagery] toggling an overlay layer by clicking label toggles for first imagery
#### Summary
If you have multiple imagery, and if you toggle an overlay layer by clicking the label, not the checkbox, then the layer toggles for the first image view, not the one you are toggling.

#### Expected vs Current Behavior
You should toggle the layer for the image the controls are in.

#### Steps to Reproduce
1. Create a display layout
2. Add multiple imagery views to the layout
3. hover over second image and click the layers dropdown
4. in the layers menu, click the label, not the checkbox, for a layer
5. observe that the layer appears over the first image, not the second one

#### Environment

* Open MCT Version: 
* Deployment Type: 
* OS:
* Browser:

#### Impact Check List

- [ ] Data loss or misrepresented data?
- [ ] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [ ] Does this impact a critical component?
- [ ] Is this just a visual bug with no functional impact?
- [ ] Does this block the execution of e2e tests?
- [ ] Does this have an impact on Performance?

### PR DESCRIPTION
**Pull request by davetsay**
Closes #6709

### Describe your changes:
Removed ids from dom elements and scoped labels to the inputs by wrapping inputs in label elements.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [x] Changes appear to address issue?
* [x] Reviewer has tested changes by following the provided instructions?
* [x] Changes appear not to be breaking changes?
* [x] Appropriate automated tests included?
* [x] Code style and in-line documentation are appropriate?
* [x] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [x] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

**Comment by akhenry** [comment]
> Removed ids from dom elements and scoped labels to the inputs by wrapping inputs in label elements.

Oh  _that's_ what was going on. I saw the behavior and assumed state was being tracked on a shared object.

File: e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js [hunk 1]
```diff
@@ -315,9 +296,45 @@ test.describe('Example Imagery in Display Layout', () => {
...
+        await expect(thumbsWrapperLocator).not.toHaveClass(/is-small-thumbs/);
+    });
+
+    /**
+     * Toggle layer visibility checkbox by clicking on checkbox label
+     * - should toggle checkbox and layer visibility for that image view
+     * - should NOT toggle checkbox and layer visibity for the first image view in display
+     */
+    test('Toggle layer visibility by clicking on label', async ({ page }) => {
+        await createImageryView(page);
```
Comment (ozyx) [inline e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js:309]: Let's add an annotation referencing the issue# at the top of this regression test:

```suggestion
        test.info().annotations.push({
          type: 'issue',
          description: 'https://github.com/nasa/openmct/issues/6709'
        });
        await createImageryView(page);
```

File: e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js [hunk 2]
```diff
@@ -315,9 +296,45 @@ test.describe('Example Imagery in Display Layout', () => {
...
+        await page.goto(displayLayout.url);
+
+        const imageElements = await page.locator('.c-imagery__main-image-wrapper');
+
+        await expect(imageElements).toHaveCount(2);
+
+        const imageOne = await page.locator('.c-imagery__main-image-wrapper').nth(0);
```
Comment (ozyx) [inline e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js:316]: Nice work with these locators! Small nitpick, `.locator()` is synchronous so we can remove the `await` from each of these calls.

File: src/plugins/imagery/components/LayerSettings.vue [hunk 3]
```diff
@@ -25,24 +25,23 @@
...
-        </li>
-      </ul>
+        <ul
+            @click="$event.stopPropagation()"
+        >
+            <li
+                v-for="(layer, index) in layers"
+                :key="index"
+            >
+                <label>
```
Comment (ozyx) [inline src/plugins/imagery/components/LayerSettings.vue:35]: 👏👏👏

**Review by ozyx — APPROVED** [review]

**Pull request merged**